Terminal.Gui/ConsoleDrivers/CursesDriver/binding.cs

```diff
@@ -83,6 +83,9 @@ public struct MouseEvent {
 		static void LoadMethods ()
 		{
 			var libs = UnmanagedLibrary.IsMacOSPlatform ? new string [] { "libncurses.dylib" } : new string [] { "libncursesw.so.6", "libncursesw.so.5" };
+			if (Microsoft.DotNet.PlatformAbstractions.RuntimeEnvironment.OperatingSystem.Contains ("opensuse")) {
```

Comment:
What will happen when OpenSuse fixes the problem with ncurses6? 

Is there some way of detecting this issue that doesn't rely on such a brute force check?

Reply:
You are absolutely right. Fixed with the https://github.com/gui-cs/Terminal.Gui/pull/2749/commits/32a224627c2e49eea6c3438790ae5141154adb42.

Reply:
> Is there some way of detecting this issue that doesn't rely on such a brute force check?

I really don't know. Perhaps yes or not, I'm not a very expertise on Linux. This was the workaround I found to fix it.